test/test_triggers.py

```diff
@@ -51,5 +51,18 @@ def test_last_modified(self):
         row = self.select('od_structure_part', obj_id)
         assert last_mod != row['last_modification'], "Last modification not set on update of child table (still {})".format(row['last_modification'])
 
+    def test_identifier(self):
+        row = {
+                'identifier': '',
+                'level': decimal.Decimal('50.000')
+        }
+
+        obj_id = self.insert_check('vw_cover', row)
+
+        row = self.select('od_structure_part', obj_id)
+
+        identifier = row['identifier']
+        assert identifier is not None, "Identifier not set on insert"
```

Comment:
Shouldn't it also check that it's not empty? At least there's an empty string specified above which should be replaced by the obj_id and here the check is for None.

Does

    assert identifier, "Identifier not set on insert"

work? In python a string converted to a boolean is False if it's empty or None IIRC.